src/main/java/Legin.java

```diff
@@ -0,0 +1,125 @@
+import java.util.Scanner;
+
+public class Legin {
+    private static Task[] tasks = new Task[100];
+    private static int currentTaskCount = 0;
+
+    public static void horizontalLine() {
+        System.out.println("______________________" +
+                "______________________________________");
+    }
+    public static void greet() {
+        horizontalLine();
+        System.out.println("Hello! I'm Legin, your best online companion!");
+        System.out.println("What can I do for you today my friend :D");
+        horizontalLine();
+    }
+    public static void bye() {
+        horizontalLine();
+        System.out.println("Bye " +
+            Character.toString(0x1F44B) +
+            ". Hope to see you again really soon! " +
+            Character.toString(0x1F608));
+        horizontalLine();
+    }
+
+    private static void printAddedTaskMessage() {
+        horizontalLine();
+        System.out.println("Got it. I've added this task:");
+        System.out.println(tasks[currentTaskCount]);
+        currentTaskCount++;
+        System.out.println("Now you have " + currentTaskCount + " tasks in the list.");
+        horizontalLine();
+    }
```

Comment:
good job doing this as it prevents repeated lines of code! I see that you call this quite often below :)